contracts/payloads/IGP96/PayloadIGP96.sol

```diff
@@ -63,43 +69,59 @@ contract PayloadIGP95 is PayloadIGPMain {
 
     // @notice Action 1: Set launch limits for sUSDe-GHO/USDC-GHO T4 vault
     function action1() internal isActionSkippable(1) {
+        address sUSDe_GHO_DEX = getDexAddress(33);
         {
-            //sUSDe-GHO / USDC-GHO
-            address sUSDe_GHO_DEX_ADDRESS = getDexAddress(33);
-            address USDC_GHO_DEX_ADDRESS = getDexAddress(4);
-            address sUSDe_GHO__USDC_GHO_VAULT_ADDRESS = getVaultAddress(125);
-
+            // sUSDe-GHO DEX
             {
-                // Update sUSDe-GHO<>USDC-GHO vault borrow shares limit
-                IFluidAdminDex.UserBorrowConfig[]
-                    memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
-                config_[0] = IFluidAdminDex.UserBorrowConfig({
-                    user: sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
-                    expandPercent: 30 * 1e2, // 30%
-                    expandDuration: 6 hours, // 6 hours
-                    baseDebtCeiling: 4_000_000 * 1e18, // 4M shares ($8M)
-                    maxDebtCeiling: 5_000_000 * 1e18 // 5M shares ($10M)
+                // sUSDe-GHO Dex
+                DexConfig memory DEX_sUSDe_GHO = DexConfig({
+                    dex: sUSDe_GHO_DEX,
+                    tokenA: sUSDe_ADDRESS,
+                    tokenB: GHO_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
                 });
+                setDexLimits(DEX_sUSDe_GHO); // Smart Collateral
 
-                IFluidDex(USDC_GHO_DEX_ADDRESS).updateUserBorrowConfigs(
-                    config_
-                );
+                DEX_FACTORY.setDexAuth(sUSDe_GHO_DEX, TEAM_MULTISIG, false);
             }
+        }
+        //sUSDe-GHO / USDC-GHO
+        address sUSDe_GHO_DEX_ADDRESS = getDexAddress(33);
```

Comment:
duplicate code here, let's clean up use either sUSDe_GHO_DEX from above or put the blocks into encapsulated {}